src/map/ai/controllers/pet_controller.cpp

```diff
@@ -27,7 +27,7 @@ along with this program.  If not, see http://www.gnu.org/licenses/
 #include "status_effect_container.h"
 #include "utils/petutils.h"
 
-CPetController::CPetController(CPetEntity* _PPet)
+CPetController::CPetController(CMobEntity* _PPet)
```

Comment:
So, a mob's pet is still a mob entity, but is using a pet controller? But a player's pet is a pet entity that uses a pet controller?

Reply:
And a players pet can also be a mob entity that uses a pet controller, yeah.